model/table/TraceVariableDataProvider.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace oat\taoOutcomeUi\model\table;
+
+use common_Logger;
+use core_kernel_classes_Resource;
+use oat\taoOutcomeUi\model\ResultsService;
+use tao_helpers_Date;
+use tao_models_classes_table_Column;
+use tao_models_classes_table_DataProvider;
+
+class TraceVariableDataProvider implements tao_models_classes_table_DataProvider
+{
+    const PROP_TRACE_VARIABLE = 'trace_variable';
```

Comment:
Maybe we can explicitly add here a proper access modifier:

`public const PROP_TRACE_VARIABLE = 'trace_variable';`

Reply:
Sure. Forgot it (